src/activities/moadmmh_activity/Moadmmh_activity.qml

```diff
@@ -0,0 +1,130 @@
+/* GCompris - moadmmh_activity.qml
+ *
+ * Copyright (C) 2018 YOUR NAME <xx@yy.org>
+ *
+ * Authors:
+ *   <THE GTK VERSION AUTHOR> (GTK+ version)
+ *   Moad Mohammed Elhebri <moadmohammedelhebri@gmail.com> (Qt Quick port)
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <https://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.6
+import GCompris 1.0
+
+import "../../core"
+import "moadmmh_activity.js" as Activity
+
+ActivityBase {
+    id: activity
+
+    onStart: focus = true
+    onStop: {}
+
+    pageComponent: Image {
+        id: background
+        source: Activity.resourceURL + "moadmmh-bg.svg"
+        sourceSize.height: background.height
+        fillMode: Image.PreserveAspectCrop
+        anchors.horizontalCenter: parent.horizontalCenter
+
+
+        signal start
+        signal stop
+
+        Component.onCompleted: {
+            activity.start.connect(start)
+            activity.stop.connect(stop)
+        }
+
+        // Add here the QML items you need to access in javascript
+
+        QtObject {
+            id: items
+            property Item main: activity.main
+            property alias background: background
+            property alias rectangle: rectangle
+            property alias bar: bar
+            property alias bonus: bonus
+        }
+
+        onStart: { Activity.start(items) }
+        onStop: { Activity.stop() }
+
+        Rectangle {
+            id: rectangle
+            width: 300; height: 300
```

Comment:
be careful to not hardcode dimensions/positions/sizes as GCompris is available on multiple different supports like phones, tablets and computers.